Plugin.Maui.ScreenSecurity/Platforms/iOS/Helpers/IOSHelpers.cs

```diff
@@ -6,7 +6,25 @@ internal static class IOSHelpers
 {
     internal static UIWindow? GetWindow()
     {
-        if (UIDevice.CurrentDevice.CheckSystemVersion(15, 0))
+        if (UIDevice.CurrentDevice.CheckSystemVersion(16, 0))
+        {
+            var windowScene =  UIApplication.SharedApplication.ConnectedScenes
+                .OfType<UIWindowScene>()
+                .FirstOrDefault();
+
+            var window = windowScene.KeyWindow;
+            if (window == null)
+            {
+                window = windowScene.Windows.FirstOrDefault(w => w.IsKeyWindow);
+            }
+            if (window == null)
+            {
+                window = windowScene.Windows.FirstOrDefault();
+            }
+
```

Comment:
Please, use null-conditional operators instead of calling `window == null` twice.

Reply:
I think this should do it...